data/json/bionics.json

```diff
@@ -434,11 +707,18 @@
         "type": "bionic",
         "id": "bio_razors",
         "name": "Fingertip Razors",
+        "occupied_bodyparts": [
+            [ "HAND_L", 2 ],
+            [ "HAND_R", 2 ]
+        ],
         "description": "You possess razor-sharp retractable claws underneath your fingernails, ten double-edged blades four centimeters in length that do a small amount of unarmed slashing damage whenever your fingertips are uncovered."
     },{
         "type": "bionic",
         "id": "bio_cqb",
         "name": "Close Quarters Battle",
+        "occupied_bodyparts": [
+            [ "HEAD", 4 ]
```

Comment:
Optional: It would make sense for CQB to have sensors all over the body - controlling a human-shaped body isn't easy.